opendevin/core/exceptions.py

```diff
@@ -71,12 +66,22 @@ def __init__(
         super().__init__(message)
 
 
-# These exceptions get sent back to the LLM
-class AgentMalformedActionError(Exception):
+# This exception gets sent back to the LLM
+# It might be malformed JSON
+class LLMMalformedActionError(Exception):
     def __init__(self, message='Malformed response'):
         super().__init__(message)
 
 
-class AgentNoActionError(Exception):
+# This exception gets sent back to the LLM
+# For some reason, the agent did not return an action
+class LLMNoActionError(Exception):
     def __init__(self, message='Agent must return an action'):
         super().__init__(message)
+
+
+# This exception gets sent back to the LLM
+# The LLM output did not include an action, or the action was not the expected type
+class LLMOutputError(Exception):
```

Comment:
The "output" made me still think: what direction.
Like "`LLMResponseError`" sounds clearer to me.

Reply:
TBH now I'm not sure we need both LLMMalformedActionError and LLMResponseError 😅 What's in a name, hah.